tests/e2e.spec.ts

```diff
@@ -0,0 +1,149 @@
+import { test, expect, Page } from '@playwright/test';
+
+const QUIZ_TITLE = 'Solvro Rulez';
```

Comment:
ogólnie ja lubię dla testowych danych dawać jakieś prefiksy, żeby potem móc łatwo je kasować z bazki np.

```suggestion
const QUIZ_TITLE = 'E2E_TESTING_QUIZ_Solvro Rulez';
```